dev/tasks/tasks.yml

```diff
@@ -1106,6 +1106,15 @@ tasks:
     template: r/github.linux.offline.build.yml
 
 
+  test-r-rhub-debian-gcc-release-ccache:
+    ci: azure
+    template: r/azure.linux.yml
+    params:
+      r_org: rhub
+      r_image: debian-gcc-release
+      r_tag: latest
+      r_build_ccache: true
```

Comment:
Do we really need an additional CI build test ccache? Can you just add `r_build_ccache` to an existing build?

Reply:
Yes, we might be able to — I had it separated to try and isolate / replicate and then once it is fixed see if we can reduce it to another one. And on that topic — do you have any idea what's going on with this? It looks like it might be quoting / not quoting properly in our cmake or (a? some?) of our dependencies.

Reply:
Here's the relevant bit from the logs:

```
[  1%] Creating directories for 'zlib_ep'
[  1%] Creating directories for 'aws_c_common_ep'
[  1%] Performing download step (download, verify and extract) for 'zlib_ep'
[  2%] Performing download step (download, verify and extract) for 'aws_c_common_ep'
-- aws_c_common_ep download command succeeded.  See also /tmp/Rtmp2ClZgX/file9b726ce22/aws_c_common_ep-prefix/src/aws_c_common_ep-stamp/aws_c_common_ep-download-*.log
[  2%] No update step for 'aws_c_common_ep'
[  3%] No patch step for 'aws_c_common_ep'
[  3%] Performing configure step for 'aws_c_common_ep'
-- zlib_ep download command succeeded.  See also /tmp/Rtmp2ClZgX/file9b726ce22/zlib_ep-prefix/src/zlib_ep-stamp/zlib_ep-download-*.log
[  4%] No update step for 'zlib_ep'
[  4%] No patch step for 'zlib_ep'
[  5%] Performing configure step for 'zlib_ep'
-- zlib_ep configure command succeeded.  See also /tmp/Rtmp2ClZgX/file9b726ce22/zlib_ep-prefix/src/zlib_ep-stamp/zlib_ep-configure-*.log
[  5%] Performing build step for 'zlib_ep'
CMake Error at /tmp/Rtmp2ClZgX/file9b726ce22/zlib_ep-prefix/src/zlib_ep-stamp/zlib_ep-build-RELEASE.cmake:37 (message):
  Command failed: 2

   'make'

  See also

    /tmp/Rtmp2ClZgX/file9b726ce22/zlib_ep-prefix/src/zlib_ep-stamp/zlib_ep-build-*.log


-- stdout output is:
Scanning dependencies of target zlib
[  2%] Building C object CMakeFiles/zlib.dir/adler32.o

-- stderr output is:
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/zlib.dir/build.make:82: CMakeFiles/zlib.dir/adler32.o] Error 1
make[4]: *** [CMakeFiles/Makefile2:182: CMakeFiles/zlib.dir/all] Error 2
make[3]: *** [Makefile:160: all] Error 2

CMake Error at /tmp/Rtmp2ClZgX/file9b726ce22/zlib_ep-prefix/src/zlib_ep-stamp/zlib_ep-build-RELEASE.cmake:47 (message):
  Stopping after outputting logs.


make[2]: *** [CMakeFiles/zlib_ep.dir/build.make:131: zlib_ep-prefix/src/zlib_ep-stamp/zlib_ep-build] Error 1
make[1]: *** [CMakeFiles/Makefile2:1395: CMakeFiles/zlib_ep.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
-- aws_c_common_ep configure command succeeded.  See also /tmp/Rtmp2ClZgX/file9b726ce22/aws_c_common_ep-prefix/src/aws_c_common_ep-stamp/aws_c_common_ep-configure-*.log
[  6%] Performing build step for 'aws_c_common_ep'
CMake Error at /tmp/Rtmp2ClZgX/file9b726ce22/aws_c_common_ep-prefix/src/aws_c_common_ep-stamp/aws_c_common_ep-build-RELEASE.cmake:37 (message):
  Command failed: 2

   'make'

  See also

    /tmp/Rtmp2ClZgX/file9b726ce22/aws_c_common_ep-prefix/src/aws_c_common_ep-stamp/aws_c_common_ep-build-*.log


-- stdout output is:
Scanning dependencies of target aws-c-common
[  0%] Building C object CMakeFiles/aws-c-common.dir/source/allocator.c.o
[  1%] Building C object CMakeFiles/aws-c-common.dir/source/allocator_sba.c.o

-- stderr output is:
/usr/bin/ccache: invalid option -- 'D'
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/aws-c-common.dir/build.make:95: CMakeFiles/aws-c-common.dir/source/allocator_sba.c.o] Error 1
make[5]: *** Waiting for unfinished jobs....
make[5]: *** [CMakeFiles/aws-c-common.dir/build.make:82: CMakeFiles/aws-c-common.dir/source/allocator.c.o] Error 1
make[4]: *** [CMakeFiles/Makefile2:145: CMakeFiles/aws-c-common.dir/all] Error 2
make[3]: *** [Makefile:160: all] Error 2

CMake Error at /tmp/Rtmp2ClZgX/file9b726ce22/aws_c_common_ep-prefix/src/aws_c_common_ep-stamp/aws_c_common_ep-build-RELEASE.cmake:47 (message):
  Stopping after outputting logs.


make[2]: *** [CMakeFiles/aws_c_common_ep.dir/build.make:131: aws_c_common_ep-prefix/src/aws_c_common_ep-stamp/aws_c_common_ep-build] Error 1
make[1]: *** [CMakeFiles/Makefile2:814: CMakeFiles/aws_c_common_ep.dir/all] Error 2
gmake: *** [Makefile:160: all] Error 2
**** Error building Arrow C++.  
```

Reply:
Do you know what the full `ccache` command looks like?

Reply:
It'll be something like `ccache g++ ...` I can run the build with verbosity to get the full full command (though it'll be after I'm done doing the same for a different CI build...)

Reply:
Here's the output with thirdparty verbosity on:

```
[ 21%] No update step for 'zstd_ep'
[ 21%] No patch step for 'zstd_ep'
[ 22%] Performing configure step for 'zstd_ep'
CMake Error at /tmp/RtmpHwiGgY/fileb65287adfe/jemalloc_ep-prefix/src/jemalloc_ep-stamp/jemalloc_ep-configure-RELEASE.cmake:37 (message):
  Command failed: 77

   './configure' 'AR=/usr/bin/ar' 'CC=/usr/bin/ccache' '--prefix=/tmp/RtmpHwiGgY/fileb65287adfe/jemalloc_ep-prefix/src/jemalloc_ep/dist/' '--libdir=/tmp/RtmpHwiGgY/fileb65287adfe/jemalloc_ep-prefix/src/jemalloc_ep/dist//lib' '--with-jemalloc-prefix=je_arrow_' '--with-private-namespace=je_arrow_private_' '--without-export' '--disable-shared' '--disable-cxx' '--disable-libdl' '--disable-initial-exec-tls'

  See also

    /tmp/RtmpHwiGgY/fileb65287adfe/jemalloc_ep-prefix/src/jemalloc_ep-stamp/jemalloc_ep-configure-*.log


-- stdout output is:
checking for xsltproc... false
checking for gcc... /usr/bin/ccache
checking whether the C compiler works... no

-- stderr output is:
configure: error: in `/tmp/RtmpHwiGgY/fileb65287adfe/jemalloc_ep-prefix/src/jemalloc_ep':
configure: error: C compiler cannot create executables
See `config.log' for more details

CMake Error at /tmp/RtmpHwiGgY/fileb65287adfe/jemalloc_ep-prefix/src/jemalloc_ep-stamp/jemalloc_ep-configure-RELEASE.cmake:47 (message):
  Stopping after outputting logs.


make[2]: *** [CMakeFiles/jemalloc_ep.dir/build.make:126: jemalloc_ep-prefix/src/jemalloc_ep-stamp/jemalloc_ep-configure] Error 1
make[1]: *** [CMakeFiles/Makefile2:841: CMakeFiles/jemalloc_ep.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
-- rapidjson_ep configure command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/src/rapidjson_ep-stamp/rapidjson_ep-configure-*.log
[ 23%] Performing build step for 'rapidjson_ep'
-- rapidjson_ep build command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/src/rapidjson_ep-stamp/rapidjson_ep-build-*.log
[ 24%] Performing install step for 'rapidjson_ep'
-- rapidjson_ep install command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/src/rapidjson_ep-stamp/rapidjson_ep-install-*.log
[ 24%] Completed 'rapidjson_ep'
[ 24%] Built target rapidjson_ep
-- brotli_ep configure command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/brotli_ep-prefix/src/brotli_ep-stamp/brotli_ep-configure-*.log
[ 24%] Performing build step for 'brotli_ep'
CMake Error at /tmp/RtmpHwiGgY/fileb65287adfe/brotli_ep-prefix/src/brotli_ep-stamp/brotli_ep-build-RELEASE.cmake:37 (message):
  Command failed: 2

   'make'

  See also

    /tmp/RtmpHwiGgY/fileb65287adfe/brotli_ep-prefix/src/brotli_ep-stamp/brotli_ep-build-*.log


-- stdout output is:
Scanning dependencies of target brotlicommon
Scanning dependencies of target brotlicommon-static
[  1%] Building C object CMakeFiles/brotlicommon-static.dir/c/common/constants.c.o
[  3%] Building C object CMakeFiles/brotlicommon-static.dir/c/common/context.c.o
[  4%] Building C object CMakeFiles/brotlicommon.dir/c/common/constants.c.o
[  6%] Building C object CMakeFiles/brotlicommon.dir/c/common/context.c.o
[  7%] Building C object CMakeFiles/brotlicommon.dir/c/common/dictionary.c.o

-- stderr output is:
/usr/bin/ccache: invalid option -- 'D'
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/brotlicommon-static.dir/build.make:82: CMakeFiles/brotlicommon-static.dir/c/common/constants.c.o] Error 1
make[5]: *** Waiting for unfinished jobs....
make[5]: *** [CMakeFiles/brotlicommon-static.dir/build.make:95: CMakeFiles/brotlicommon-static.dir/c/common/context.c.o] Error 1
make[4]: *** [CMakeFiles/Makefile2:785: CMakeFiles/brotlicommon-static.dir/all] Error 2
make[4]: *** Waiting for unfinished jobs....
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/brotlicommon.dir/build.make:82: CMakeFiles/brotlicommon.dir/c/common/constants.c.o] Error 1
make[5]: *** Waiting for unfinished jobs....
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/brotlicommon.dir/build.make:95: CMakeFiles/brotlicommon.dir/c/common/context.c.o] Error 1
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/brotlicommon.dir/build.make:108: CMakeFiles/brotlicommon.dir/c/common/dictionary.c.o] Error 1
make[4]: *** [CMakeFiles/Makefile2:731: CMakeFiles/brotlicommon.dir/all] Error 2
make[3]: *** [Makefile:160: all] Error 2

CMake Error at /tmp/RtmpHwiGgY/fileb65287adfe/brotli_ep-prefix/src/brotli_ep-stamp/brotli_ep-build-RELEASE.cmake:47 (message):
  Stopping after outputting logs.


make[2]: *** [CMakeFiles/brotli_ep.dir/build.make:131: brotli_ep-prefix/src/brotli_ep-stamp/brotli_ep-build] Error 1
make[1]: *** [CMakeFiles/Makefile2:1112: CMakeFiles/brotli_ep.dir/all] Error 2
-- zlib_ep configure command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/zlib_ep-prefix/src/zlib_ep-stamp/zlib_ep-configure-*.log
[ 24%] Performing build step for 'zlib_ep'
CMake Error at /tmp/RtmpHwiGgY/fileb65287adfe/zlib_ep-prefix/src/zlib_ep-stamp/zlib_ep-build-RELEASE.cmake:37 (message):
  Command failed: 2

   'make'

  See also

    /tmp/RtmpHwiGgY/fileb65287adfe/zlib_ep-prefix/src/zlib_ep-stamp/zlib_ep-build-*.log


-- stdout output is:
Scanning dependencies of target zlibstatic
Scanning dependencies of target zlib
[  2%] Building C object CMakeFiles/zlibstatic.dir/compress.o
[  5%] Building C object CMakeFiles/zlib.dir/adler32.o
[  8%] Building C object CMakeFiles/zlibstatic.dir/adler32.o
[ 11%] Building C object CMakeFiles/zlib.dir/compress.o
[ 13%] Building C object CMakeFiles/zlib.dir/crc32.o
[ 16%] Building C object CMakeFiles/zlibstatic.dir/crc32.o

-- stderr output is:
/usr/bin/ccache: invalid option -- 'I'
make[5]: *** [CMakeFiles/zlibstatic.dir/build.make:95: CMakeFiles/zlibstatic.dir/compress.o] Error 1
make[5]: *** Waiting for unfinished jobs....
/usr/bin/ccache: invalid option -- 'I'
make[5]: *** [CMakeFiles/zlibstatic.dir/build.make:82: CMakeFiles/zlibstatic.dir/adler32.o] Error 1
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/zlib.dir/build.make:82: CMakeFiles/zlib.dir/adler32.o] Error 1
make[5]: *** Waiting for unfinished jobs....
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/zlib.dir/build.make:95: CMakeFiles/zlib.dir/compress.o] Error 1
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/zlib.dir/build.make:108: CMakeFiles/zlib.dir/crc32.o] Error 1
make[4]: *** [CMakeFiles/Makefile2:182: CMakeFiles/zlib.dir/all] Error 2
make[4]: *** Waiting for unfinished jobs....
/usr/bin/ccache: invalid option -- 'I'
make[5]: *** [CMakeFiles/zlibstatic.dir/build.make:108: CMakeFiles/zlibstatic.dir/crc32.o] Error 1
make[4]: *** [CMakeFiles/Makefile2:155: CMakeFiles/zlibstatic.dir/all] Error 2
make[3]: *** [Makefile:160: all] Error 2

CMake Error at /tmp/RtmpHwiGgY/fileb65287adfe/zlib_ep-prefix/src/zlib_ep-stamp/zlib_ep-build-RELEASE.cmake:47 (message):
  Stopping after outputting logs.


make[2]: *** [CMakeFiles/zlib_ep.dir/build.make:131: zlib_ep-prefix/src/zlib_ep-stamp/zlib_ep-build] Error 1
make[1]: *** [CMakeFiles/Makefile2:1395: CMakeFiles/zlib_ep.dir/all] Error 2
-- boost_ep download command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/boost_ep-prefix/src/boost_ep-stamp/boost_ep-download-*.log
[ 25%] No update step for 'boost_ep'
[ 25%] No patch step for 'boost_ep'
-- zstd_ep configure command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep-stamp/zstd_ep-configure-*.log
[ 26%] No configure step for 'boost_ep'
[ 27%] Performing build step for 'zstd_ep'
[ 27%] No build step for 'boost_ep'
[ 27%] No install step for 'boost_ep'
[ 28%] Completed 'boost_ep'
-- aws_c_common_ep configure command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/aws_c_common_ep-prefix/src/aws_c_common_ep-stamp/aws_c_common_ep-configure-*.log
[ 29%] Performing build step for 'aws_c_common_ep'
[ 29%] Built target boost_ep
CMake Error at /tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep-stamp/zstd_ep-build-RELEASE.cmake:37 (message):
  Command failed: 2

   'make'

  See also

    /tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep-stamp/zstd_ep-build-*.log


-- stdout output is:
Scanning dependencies of target libzstd_static
[  3%] Building C object lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/debug.c.o
[ 10%] Building C object lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/entropy_common.c.o
[ 10%] Building C object lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/error_private.c.o
[ 13%] Building C object lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/fse_decompress.c.o
[ 16%] Building C object lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/pool.c.o
[ 20%] Building C object lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/threading.c.o

-- stderr output is:
/usr/bin/ccache: invalid option -- 'D'
/usr/bin/ccache: invalid option -- 'D'
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [lib/CMakeFiles/libzstd_static.dir/build.make:82: lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/debug.c.o] Error 1
make[5]: *** Waiting for unfinished jobs....
make[5]: *** [lib/CMakeFiles/libzstd_static.dir/build.make:108: lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/error_private.c.o] Error 1
/usr/bin/ccache: invalid option -- 'D'
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [lib/CMakeFiles/libzstd_static.dir/build.make:121: lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/fse_decompress.c.o] Error 1
make[5]: *** [lib/CMakeFiles/libzstd_static.dir/build.make:134: lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/pool.c.o] Error 1
make[5]: *** [lib/CMakeFiles/libzstd_static.dir/build.make:95: lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/entropy_common.c.o] Error 1
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [lib/CMakeFiles/libzstd_static.dir/build.make:147: lib/CMakeFiles/libzstd_static.dir/tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep/lib/common/threading.c.o] Error 1
make[4]: *** [CMakeFiles/Makefile2:142: lib/CMakeFiles/libzstd_static.dir/all] Error 2
make[3]: *** [Makefile:149: all] Error 2

CMake Error at /tmp/RtmpHwiGgY/fileb65287adfe/zstd_ep-prefix/src/zstd_ep-stamp/zstd_ep-build-RELEASE.cmake:47 (message):
  Stopping after outputting logs.


make[2]: *** [CMakeFiles/zstd_ep.dir/build.make:131: zstd_ep-prefix/src/zstd_ep-stamp/zstd_ep-build] Error 1
make[1]: *** [CMakeFiles/Makefile2:977: CMakeFiles/zstd_ep.dir/all] Error 2
CMake Error at /tmp/RtmpHwiGgY/fileb65287adfe/aws_c_common_ep-prefix/src/aws_c_common_ep-stamp/aws_c_common_ep-build-RELEASE.cmake:37 (message):
  Command failed: 2

   'make'

  See also

    /tmp/RtmpHwiGgY/fileb65287adfe/aws_c_common_ep-prefix/src/aws_c_common_ep-stamp/aws_c_common_ep-build-*.log


-- stdout output is:
Scanning dependencies of target aws-c-common
[  0%] Building C object CMakeFiles/aws-c-common.dir/source/allocator.c.o
[  2%] Building C object CMakeFiles/aws-c-common.dir/source/allocator_sba.c.o
[  2%] Building C object CMakeFiles/aws-c-common.dir/source/arch/generic/cpuid.c.o
[  3%] Building C object CMakeFiles/aws-c-common.dir/source/array_list.c.o
[  5%] Building C object CMakeFiles/aws-c-common.dir/source/assert.c.o
[  5%] Building C object CMakeFiles/aws-c-common.dir/source/byte_buf.c.o

-- stderr output is:
/usr/bin/ccache: invalid option -- 'D'
/usr/bin/ccache: invalid option -- 'D'
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/aws-c-common.dir/build.make:82: CMakeFiles/aws-c-common.dir/source/allocator.c.o] Error 1
make[5]: *** Waiting for unfinished jobs....
make[5]: *** [CMakeFiles/aws-c-common.dir/build.make:95: CMakeFiles/aws-c-common.dir/source/allocator_sba.c.o] Error 1
make[5]: *** [CMakeFiles/aws-c-common.dir/build.make:108: CMakeFiles/aws-c-common.dir/source/arch/generic/cpuid.c.o] Error 1
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/aws-c-common.dir/build.make:121: CMakeFiles/aws-c-common.dir/source/array_list.c.o] Error 1
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/aws-c-common.dir/build.make:134: CMakeFiles/aws-c-common.dir/source/assert.c.o] Error 1
/usr/bin/ccache: invalid option -- 'D'
make[5]: *** [CMakeFiles/aws-c-common.dir/build.make:147: CMakeFiles/aws-c-common.dir/source/byte_buf.c.o] Error 1
make[4]: *** [CMakeFiles/Makefile2:145: CMakeFiles/aws-c-common.dir/all] Error 2
make[3]: *** [Makefile:160: all] Error 2

CMake Error at /tmp/RtmpHwiGgY/fileb65287adfe/aws_c_common_ep-prefix/src/aws_c_common_ep-stamp/aws_c_common_ep-build-RELEASE.cmake:47 (message):
  Stopping after outputting logs.


make[2]: *** [CMakeFiles/aws_c_common_ep.dir/build.make:131: aws_c_common_ep-prefix/src/aws_c_common_ep-stamp/aws_c_common_ep-build] Error 1
make[1]: *** [CMakeFiles/Makefile2:814: CMakeFiles/aws_c_common_ep.dir/all] Error 2
-- snappy_ep configure command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/snappy_ep-prefix/src/snappy_ep-stamp/snappy_ep-configure-*.log
[ 30%] Performing build step for 'snappy_ep'
-- snappy_ep build command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/snappy_ep-prefix/src/snappy_ep-stamp/snappy_ep-build-*.log
[ 31%] Performing install step for 'snappy_ep'
-- snappy_ep install command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/snappy_ep-prefix/src/snappy_ep-stamp/snappy_ep-install-*.log
[ 31%] Completed 'snappy_ep'
[ 31%] Built target snappy_ep
-- bzip2_ep build command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/bzip2_ep-prefix/src/bzip2_ep-stamp/bzip2_ep-build-*.log
[ 31%] Performing install step for 'bzip2_ep'
-- bzip2_ep install command succeeded.  See also /tmp/RtmpHwiGgY/fileb65287adfe/bzip2_ep-prefix/src/bzip2_ep-stamp/bzip2_ep-install-*.log
[ 32%] Completed 'bzip2_ep'
[ 32%] Built target bzip2_ep
gmake: *** [Makefile:160: all] Error 2
**** Error building Arrow C++.  
```